projects/plugins/boost/app/modules/cache/cache.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+class Boost_Cache {
+	private $cache_key     = false;
+	private $path_key      = false;
+	protected $request_uri = false;
```

Comment:
Done. and I was able to remove $path_key since it's the same as $path in the file cache. If we have different cache backends we can put it back in, if required.